src/main/java/io/github/railroad/utility/FileHandler.java

```diff
@@ -25,4 +26,67 @@ public static void copyUrlToFile(String url, Path path) {
             throw new RuntimeException("Failed to copy URL to file", exception);
         }
     }
+
+    public static void updateKeyValuePairByLine(String key, String value, File file) throws IOException {
+        BufferedReader reader = new BufferedReader(new FileReader(file));
+        StringBuilder stringBuilder = new StringBuilder();
+        String line;
+        while ((line = reader.readLine()) != null) {
+            if (line.startsWith(key + "=")) {
+                // Replace the existing value
+                line = key + "=" + value;
+            }
+            stringBuilder.append(line).append("\n");
+        }
+        reader.close();
+        BufferedWriter writer = new BufferedWriter(new FileWriter(file));
+        writer.write(stringBuilder.toString());
+        writer.close();
+    }
```

Comment:
I'm okay with this as a temporary solution, but ideally this should be using a proper templating system rather than parsing it manually like this.